tutorials/matching_fof.ipynb

```diff
@@ -51,7 +54,7 @@
    "outputs": [],
    "source": [
     "# load object catalog (for a single tract)\n",
-    "object_cat = GCRCatalogs.load_catalog('dc2_object_run1.2i_all_columns')"
+    "object_cat = GCRCatalogs.load_catalog('dc2_object_run2.2i_dr6')"
```

Comment:
`dr6` has been removed temporarily to avoid confusion. Use `dr6a` here.

```suggestion
    "object_cat = GCRCatalogs.load_catalog('dc2_object_run2.2i_dr6a')"
```